src/en/azcomic/src/eu/kanade/tachiyomi/extension/en/azcomic/AzComic.kt

```diff
@@ -0,0 +1,341 @@
+package eu.kanade.tachiyomi.extension.en.azcomic
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import keiyoushi.utils.parseAs
+import keiyoushi.utils.tryParse
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class AzComic : HttpSource() {
+
+    override val name = "azComic"
+
+    override val baseUrl = "https://azcomic.com"
+
+    override val lang = "en"
+
+    override val supportsLatest = true
+
+    private val dateFormat by lazy {
+        SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.ENGLISH)
```

Comment:
Not using any locale specific patterns, so you shouldn't set a locale:
```suggestion
        SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.ROOT)
```

If timezone is know you can set that here too. Otherwise it will use the user's timezone.